content/en/docs/ops/deployment/requirements/index.md

```diff
@@ -71,15 +71,15 @@ To be part of a mesh, Kubernetes pods must satisfy the following requirements:
     Otherwise, you will need to [provide the permission](https://kubernetes.io/docs/concepts/policy/pod-security-policy/#authorizing-policies).
 
 - **Pods with app and version labels**: We recommend adding an explicit
-  `app` label and `version` label to the specification of the pods deployed using
-  a Kubernetes `Deployment`. The `app` and `version` labels add contextual information
+  `app.kubernetes.io/name` label and `app.kubernetes.io/version` label to the specification of the pods deployed using
```

Comment:
Should we keep `app` and `version` as well, just say they are the second option?

Reply:
Yes, we should mention both labels can be used, but the kubernetes ones are preferred. We should also update all the Istio samples to use the new ones.

Reply:
is there any plan to fully transition to just the Kubernetes app and revision labels?  Will users have to switch to one over the other eventually?